### PR TITLE
[Release notes] Add known issue about Alerts table and nested file.name field

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3609.

**_The same known issue summary is used in the 8.9 and 8.8.x release notes. Feel free to review the 8.9.0 release notes only, and I'll apply your feedback to the 8.8.x release notes._**

Previews:
- [8.9 known issues](https://security-docs_3644.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.9.0.html#known-issue-8.9.0)
- [8.8.x known issues](https://security-docs_3644.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.8.0.html)

### CODE REVIEW

File: docs/release-notes/8.8.asciidoc [hunk 1]
```diff
@@ -10,6 +10,46 @@
 ==== Known issues
 
 * Setting the `max_signals` value higher than the {kibana-ref}/alert-action-settings-kb.html#alert-settings[`xpack.alerting.rules.run.alerts.max`] value will lead to rule failure.
+* A UI bug can affect the Alerts table's height and make it difficult to view alerts. View the alerts in Timeline or choose one of the following workarounds to readjust the table:
+
+** Use the Alerts table **Fields** browser to remove fields until the table width is smaller than its container.
+** Adjust your OS’ zoom settings and refresh the page.
+** Zoom your browser in and then out.
```
Comment (kqualters-elastic) [inline docs/release-notes/8.8.asciidoc:17]: This should be something like 'zoom your browser in or out, and then refresh the page, issue will happen at some zoom levels and not at others.'

File: docs/release-notes/8.8.asciidoc [hunk 2]
```diff
@@ -193,6 +233,47 @@ GET .kibana*/_search
 }
 ----------------------------------
 
+* A UI bug can affect the Alerts table's height and make it difficult to view alerts. View the alerts in Timeline or choose one of the following workarounds to readjust the table:
+
+** Use the Alerts table **Fields** browser to remove fields until the table width is smaller than its container.
+** Adjust your OS’ zoom settings and refresh the page.
+** Zoom your browser in and then out.
```
Comment (kqualters-elastic) [inline docs/release-notes/8.8.asciidoc:240]: Same as above

File: docs/release-notes/8.8.asciidoc (hunk 1 above)
Comment (nastasha-solomon) [reply docs/release-notes/8.8.asciidoc:17]:
```suggestion
** Zoom your browser in and out, then refresh the page. (The row height issue happens at some zoom levels.)
```

File: docs/release-notes/8.8.asciidoc (hunk 2 above)
Comment (nastasha-solomon) [reply docs/release-notes/8.8.asciidoc]: Addressed in [14fa016](https://github.com/elastic/security-docs/pull/3644/commits/14fa016d290d8a68f16c88aa183458cfbfef3cae).

**Review by kqualters-elastic — APPROVED** [review]

**Review by joepeeples — COMMENTED** [review]
Content looks great, just added a few nits.

Overall, this is pretty extensive compared to our typical known issues, and I wonder how much of an edge case the issue itself is. It only happens in Chrome, and only with certain types of `file.name` values, and it'll (hopefully) get fixed so it won't affect any more Elastic versions. Clearly a lot of effort went into researching and documenting the issue and its workaround, so I'm sure customers will appreciate these clear, organized instructions!

File: docs/release-notes/8.8.asciidoc [hunk 3]
```diff
@@ -10,6 +10,50 @@
...
+**Case #1:** You want to display threat indicator match alerts with nested `file.name` fields and the Alerts table won't render. To fix this, manually edit your browser's local storage and refresh the Alerts page:
+
++
+NOTE: These instructions only apply to the Google Chrome browser. Modify the steps based on of browser you're using.
++
+
+. Right-click anywhere on the Alerts page, then select *Inspect* to open Chrome's Developer Tools.
+. Go to *Application -> Storage*, then expand *Local Storage*. 
+. Click on the name of your Kibana instance, for example, http://localhost:1234. 
+. Search for the `detection-engine-alert-table-securitySolution-rule-details-gridView` key and copy its value.
```
Comment (nastasha-solomon) [inline docs/release-notes/8.8.asciidoc:39]:
```suggestion
. Search for the `detection-engine-alert-table-securitySolution-rule-details-gridView` key and copy its value. The value you copied is the JSON blob that's used to persist the Alerts table's state, including the table's selected columns.
```

File: docs/release-notes/8.8.asciidoc [hunk 4]
```diff
@@ -10,6 +10,50 @@
...
++
+NOTE: These instructions only apply to the Google Chrome browser. Modify the steps based on of browser you're using.
++
+
+. Inspect the Alerts page to open DevTools.
+. Go to *Application -> Storage*, then expand *Local Storage*. 
+. Click on the name of your Kibana instance, for example, http://localhost:1234. 
+. Search for the `detection-engine-alert-table-securitySolution-rule-details-gridView` key and copy its value.
++
+NOTE: The value you copied is the JSON blob that's used to persist the Alerts table's state, including the table's selected columns.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/release-notes/8.8.asciidoc:41]:
```suggestion
```

**Pull request merged**